SampleBrowser/SampleBrowser/UI/Browser/BrowserViewController.swift

```diff
@@ -158,6 +181,10 @@ class BrowserViewController: UIViewController,
                                                     canGoForward: canGoForward)
     }
 
+    func didLoad(pageMetadata: EnginePageMetadata) {
+        // TODO: Handle page metadata [?]
```

Comment:
Suggestion; I would change comment to
```suggestion
        // Page metadata can be used to fetch page favicons.
        // We currently do not handle favicons in SampleBrowser, so this is empty.
```